library/src/main/java/com/chuckerteam/chucker/internal/ui/MainViewModel.kt

```diff
@@ -35,6 +42,22 @@ internal class MainViewModel : ViewModel() {
     val throwables: LiveData<List<RecordedThrowableTuple>> = RepositoryProvider.throwable()
         .getSortedThrowablesTuples()
 
+    fun getAllTransactions(): List<HttpTransaction>? = runBlocking {
+        val transactions = viewModelScope.async(Dispatchers.Default) {
+            RepositoryProvider.transaction().getAllTransactions()
+        }
+        transactions.await()
+    }
+
+    fun createExportFile(content: String, fileFactory: FileFactory): File = runBlocking {
```

Comment:
Same as above about `runBlocking`

Reply:
updated, with the same approach as above.